test/integration/scheduler/serving_test.go

```diff
@@ -0,0 +1,327 @@
+/*
```

Comment:
try to integrate into test/integration/controllermanager/serving_test.go, and rename that to test/integration/serving.